### PR TITLE
Events

### PR DESCRIPTION
**Pull request by franclobo**
### Events.
- Copy the files calculate.js and operate.js into a logic/ directory.
- Add [big.js](https://www.npmjs.com/package/big-js) as dependency.
- Import the files in the Calculator component.
- Implement the event handlers needed to use the math logic from calculate.js and operate.js

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,48 +1,130 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import calculate from '../logic/caculate';
```
Comment (BenMukebo) [inline src/components/calculator.js:3]: - [ OPTIONAL] Let's capitalize the name of the `Calculator.js` component so as to differentiate it from the imported one `calculate.js`  file from the logic directory and remember to always capitalize your react name component file
- eg: `Navbar.js, Home.js....`

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @franclobo,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

Your Pull Request has a proper title and description. ✔️   
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x] Kindly, Add a descriptive README file to your project,  A professional README file should have a project title, project description, list of technology used in the project, and instructions to set and run the project up locally, This includes `git clone URL` , `cd project_name`,  `npm install`, `npm start`, etc
Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/caculate.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
```
Comment (BenMukebo) [inline src/logic/caculate.js:1]: - [ ] Great job so far, unfortunately, I have trouble running your application and I see the result in the browser even after running `npm install and npm start`, instead I'm getting this error message 👇🏻 Kindly fix this 🥲 

- Console:
errormessage

- Brower:

messbrow

**Review by BenMukebo — CHANGES_REQUESTED** [review]

File: src/components/calculator.js (hunk 1 above)
Comment (franclobo) [reply src/components/calculator.js:3]: Done.

**Review by KDlamini — APPROVED** [review]
### Hi @franclobo 🙋‍♀️,  

Great job! 💯 
You have done a great job implementing all the required changes. 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**